devel/ci/integration/tests/test_bodhi_tasks.py

```diff
@@ -35,17 +35,34 @@ def test_push_composer_start(bodhi_container, db_container, rabbitmq_container):
     output = "".join(line.decode("utf-8") for line in output)
     assert output.endswith("Requesting a compose\n")
     # Check how many composes should be done
-    query = "SELECT COUNT(*) FROM composes"
+    # because we later check for pungi.log, we need to see if there are composes that apply
+    # we can get these from the type field of the builds table,
+    # after joining with the composes table through updates table.
+    query = """
+      select distinct
+        c.release_id, c.request, state, b.type
+      from
+        composes as c
+      inner join updates as u on (c.release_id = u.release_id and c.request = u.request )
+      inner join builds as b on (b.update_id = u.id)
+      where b.type in ('rpm', 'module') and state <> 'failed';
+    """
     db_ip = db_container.get_IPv4s()[0]
     conn = psycopg2.connect("dbname=bodhi2 user=postgres host={}".format(db_ip))
+    print("dbname=bodhi2 user=postgres host={}".format(db_ip))
```

Comment:
This is debug print or do you have this here on purpose?

Reply:
Yes, this is a left-over debug print.